.github/workflows/benchmark_pull_request.yml

```diff
@@ -0,0 +1,64 @@
+name: benchmark_pull_request
+
+on:
+  push:
+    branches:
+      - 'hacl-star-**'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  default:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build
+          sudo apt-get install python3 python3-scipy
+
+      - name: Checkout hacl-packages
```

Comment:
suggestion: I have the feeling that adding the branch name here could he useful. We could use `Checkout hacl-packages (${{ github.head_ref }})` here (and `Checkout hacl-packages (main)` in line 25).

Reply:
done :)